src/app/partials/nav/header-static/header-static.component.html

```diff
@@ -1,25 +1,37 @@
-<header id="header-static">
-  <ul class="inline-list left-list left">
-    <li class="logo">
-      <img src="/assets/images/EvalAI_favicon.png" alt="">
-    </li>
-  </ul>
-  <ul class="inline-list right-list right right-nav">
-    <li>
-      <a class="light-link" href="/"> Challenges</a>
-    </li>
-    <li>
-      <a class="light-link" href="/"> News</a>
-    </li>
-    <li>
-      <a class="light-link" href="/"> Forum</a>
-    </li>
-    <li>
-      <a class="light-link" href="/"> Join Us</a>
-    </li>
-    <li class="pointer">
-      ...
-    </li>
-  </ul>
-  <div class="clear-fix"></div>
+<header id="header-static" [class.header-shadow]="headerWhite" [class.header-shadow-large]="scrolledState" [class.white-bg]="headerWhite" [class.dark-bg]="isMenuExpanded && atHome && innerWidth <= 810">
+    <div class="logo-container">
+      <img src="assets/images/EvalAI_favicon.png" class = "logo-header" [class.invert-color]="headerWhite" *ngIf="atHome" (click)="isMenuExpanded = false; navigateTo(['']);"/>
+      <img src="assets/images/evalai_logo_light.png" *ngIf="!atHome" class = "logo-header" [class.invert-color]="headerWhite" (click)="isMenuExpanded = false; navigateTo(['']);"/>
+    </div>
+    
+    <div class="menu-expander" [class.menu-change]="isMenuExpanded" (click)="menuExpander()">
+        <div class="menu-bar1" [class.dark-bg]="headerWhite" [class.white-bg]="!headerWhite" ></div>
+        <div class="menu-bar2" [class.dark-bg]="headerWhite" [class.white-bg]="!headerWhite"></div>
+        <div class="menu-bar3" [class.dark-bg]="headerWhite" [class.white-bg]="!headerWhite"></div>
+    </div>
+
+    <div class="navlinks-container" [class.height-zero]="!isMenuExpanded" [class.white-bg]="headerWhite">
+    <div class="main-wrapper clearfix">
+        <span class = "navlinks navlinktext fw-light" (click)="isMenuExpanded = false;" [class.dark-text]="headerWhite"> Challenges </span>
```

Comment:
Please use `li` instead of `span` as we have in current version. @live-wire

Reply:
Used `li`